tutorials/time-series-visualization-with-altair/tutorial.ipynb

```diff
@@ -0,0 +1,2133 @@
+{
```

Comment:
<p>Suggest adding the Sierra Nevada ecoregion as a reference layer, to connect this part of the tutorial with the prior content.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='93'/>

Reply:
<p>Done.</p>